content/en/_index.md

```diff
@@ -115,6 +132,17 @@ public static void main(String[] args) {
     exampleSnippet(projectId, filePath);
 }
 {{< /tab >}}
+{{< tab header="PHP" >}}
+/**
+ * PHP has evolved separate from the guidelines on Method Structure & No CLIs.
+ *
+ * TODO(main runner): Should PHP samples start using the main runner concept?
+ * TODO(no cli clarification): Related, PHP is doing CLIs https://github.com/GoogleCloudPlatform/php-docs-samples/blob/master/testing/sample_helpers.php
+ * TODO(parameters): There are two models of documenting parameters:
+ * - Docblock comment: https://github.com/GoogleCloudPlatform/php-docs-samples/blob/master/media/transcoder/src/list_job_templates.php#L30
+ * - Commented out variables in global scope: https://github.com/GoogleCloudPlatform/php-docs-samples/blob/master/bigquery/api/src/browse_table.php#L36
+ */
+{{< /tab >}}
```

Comment:
@bshaffer I'm having trouble parsing where PHP is diverging from the guidelines deliberately. Here is the preview rule: https://googlecloudplatform.github.io/samples-style-guide/#main-runner

Node Guide Sample in Progress: https://github.com/GoogleCloudPlatform/samples-style-guide/pull/33/files#
Ruby Guide Sample in Progress: https://github.com/GoogleCloudPlatform/samples-style-guide/pull/33

Do you have a perspective on what a good path forward is for PHP? Does it need to do something different than the style guide rule?